common/post-fs-data.sh

```diff
@@ -16,13 +17,30 @@ is_mounted() {
   return $?
 }
 
+is_mounted_rw() {
+  cat /proc/mounts | grep -q " `readlink -f $1` " | grep -q " rw," 2>/dev/null
+  return $?
+}
+
+mount_rw() {
+  mount -o remount,rw $1
+  DID_MOUNT_RW=1
+}
+
+unmount_rw() {
```

Comment:
Correct me if I'm wrong but wouldn't "x$1" be "x/product" and not "x1"?

Also what is the use off adding x before your variables?

Reply:
Other than this I'll likely accept it btw.

Reply:
Ahh I see. Seems like a good habit to have, I'll try to remember it in the future.